packages/repository/src/relations/belongs-to/belongs-to.helpers.ts

```diff
@@ -49,12 +48,24 @@ export function resolveBelongsToMetadata(relationMeta: BelongsToDefinition) {
   const targetName = targetModel.modelName;
   debug('Resolved model %s from given metadata: %o', targetName, targetModel);
 
+  let keyFrom;
+  if (
+    relationMeta.keyFrom &&
+    relationMeta.source.definition.properties[relationMeta.keyFrom]
+  ) {
+    keyFrom = relationMeta.keyFrom;
+  } else {
+    keyFrom = camelCase(targetName + '_id');
+  }
+
```

Comment:
I think the current design allows users to customize the `keyFrom` and `keyTo` value. And `keyFrom` is default to the decorated key (e.g `customerId`), see [code](https://github.com/strongloop/loopback-next/blob/master/packages/repository/src/relations/belongs-to/belongs-to.decorator.ts#L50). Even it throws when `keyFrom` is undefined, I don't think this situation would happen as it would default the decorated key anyway. So I don't think the change is necessary.

Reply:
I think the intention here is to allow users to omit `keyFrom` when calling `modelDefinition.belongsTo()` - that's a different code path from the one used by `@belongsTo` decorator. When calling `belongsTo`, I think it's useful to be able to leave out `keyFrom` and let the framework to compute it.